CMakeLists.txt

```
@@ -8,6 +8,13 @@ set (alldirs
   GEOSwgcm_GridComp
   )
 
+option(BUILD_WITH_GIGATRAJ "Build GEOSgcm with Gigatraj" ON)
```

Do we want this to default to `ON`? Might be best to initially add it as an optional feature.

---

I will make if OFF as default. The on is just for my own convenience.

---

Definitely needs to be OFF as default.
```suggestion
option(BUILD_WITH_GIGATRAJ "Build GEOSgcm with Gigatraj" OFF)
```